tokio/src/runtime/scheduler/multi_thread/worker.rs

```diff
@@ -1014,7 +1014,7 @@ impl Core {
             .tuned_global_queue_interval(&worker.handle.shared.config);
 
         // Smooth out jitter
-        if abs_diff(self.global_queue_interval, next) > 2 {
+        if self.global_queue_interval.abs_diff(next) > 2 {
```

Comment:
Hmm, I think it reads better like this:
```suggestion
        if u32::abs_diff(self.global_queue_interval, next) > 2 {
```

Reply:
alright!